content/docs/user-guide/packaging/project-export/_index.md

```diff
@@ -0,0 +1,15 @@
+---
+linkTitle: Project Export
+title: Project Export
+weight: 1600
```

Comment:
If you want this section to appear in the same order as listed in packaging/_index.md (i.e. inserted before "Asset Bundler"), then the weight here should be 450.
```suggestion
weight: 450
```